src/backend/legendary/games.ts

```diff
@@ -643,6 +643,27 @@ class LegendaryGame extends Game {
           : ['--wine', wineBin])
       )
     }
+
+    // Log any launch information configured in Legendary's config.ini
+    await runLegendaryCommand(['launch', this.appName, '--json'], {
+      onOutput: (output) => {
+        if (output.match(/game_parameters/)) {
+          appendFileSync(
+            this.logFileLocation,
+            "Legendary's base config from config.ini:\n"
```

Comment:
I'm not super sure about this line, maybe `Legendary's launch command before Heroic configuration`? or something like that